integrations/weaviate/src/haystack_integrations/components/retrievers/weaviate/bm25_retriever.py

```diff
@@ -26,6 +26,7 @@ def __init__(
         document_store: WeaviateDocumentStore,
         filters: Optional[Dict[str, Any]] = None,
         top_k: int = 10,
+        filter_policy: Literal["replace", "merge"] = "replace",
```

Comment:
This should be an enumeration in haystack core.

Reply:
It already is, I couldn't use it back in the day because it Haystack update wasn't released but now that it is - we can do everything in one shot. Thanks for pointing this out @shadeMe